spec/features/user_visits_styles_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+feature "As a user, I want to view beers by style", %{
+  [ ] I must be able to view beers by style
+} do
+  scenario "search a style with an existing beer" do
+    style = FactoryGirl.create(:style)
+    beer = FactoryGirl.create(:beer, style_id: style.id)
```

Comment:
calling `beer = FactoryGirl.create(:beer)` would give you a style of `beer.style` so you don't have to define style before this.